Gemfile

```diff
@@ -33,9 +33,6 @@ group :omnibus do
   gem "appbundler"
   gem "ed25519" # ed25519 ssh key support done here as its a native gem we can't put in the gemspec
   gem "bcrypt_pbkdf" # ed25519 ssh key support done here as its a native gem we can't put in the gemspec
-  if probably_x86?
```

Comment:
We can remove the def for `probably_x86?` as well, as it is only used here.

Reply:
Updated. Thanks @clintoncwolfe